Directory.Packages.props

```diff
@@ -2,16 +2,17 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <MsBuildPackageVersion>17.2.0</MsBuildPackageVersion>
+    <NuGetAuditMode>direct</NuGetAuditMode>
   </PropertyGroup>
   <ItemGroup>
     <PackageVersion Include="CliWrap" Version="3.6.4" />
-    <PackageVersion Include="FSharp.Core" Version="[8.0.400]" />
-    <PackageVersion Include="FSharp.Compiler.Service" Version="[43.8.400]" />
+    <PackageVersion Include="FSharp.Core" Version="[9.0.100]" />
+    <PackageVersion Include="FSharp.Compiler.Service" Version="[43.9.100]" />
     <PackageVersion Include="Ionide.KeepAChangelog.Tasks" Version="0.1.8" PrivateAssets="all" />
     <PackageVersion Include="McMaster.NETCore.Plugins" Version="1.4.0" />
     <PackageVersion Include="Argu" Version="6.1.1" />
     <PackageVersion Include="Glob" Version="1.1.9" />
-    <PackageVersion Include="Ionide.ProjInfo.ProjectSystem" Version="0.65.0" />
+    <PackageVersion Include="Ionide.ProjInfo.ProjectSystem" Version="0.67.0" />
```

Comment:
We probably need to wait until we release https://github.com/ionide/proj-info/pull/219